### PR TITLE
Return switching to new release note

### PR DESCRIPTION
**Pull request by EmilFattakhov**
This PR:

1. Adds switching to a newer release section for Pulsar.
2. Adds wiping the Pulsar section.
3. Moves wiping commands for the advanced CLI to a more meaningful spot. 

Any additional feedback is welcome, of course!

### CODE REVIEW

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 1]
```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
 
 There are some cases where version updates will cause issue with your Node & Farmer and you may have to wipe & purge your node, typically when errors occur. If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
 
+### Wipe & Purge
+
+If you were running a node previously, and want to switch to a new snapshot, please perform these steps and then follow the guideline again:
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:58]:
```suggestion
If you were running a node previously, and want to switch to a new network, please perform these steps and then follow the guide again:
```

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 2]
```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
...
+```bash
+# Replace `FARMER_FILE_NAME` with the name of the farmer file you downloaded from releases
+./FARMER_FILE_NAME wipe PATH_TO_FARM
+# Replace `NODE_FILE_NAME` with the name of the node file you downloaded from releases
+./NODE_FILE_NAME purge-chain --chain gemini-3g
+```
+Does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:65]:
```suggestion
It does not matter if the node/farmer executable is the previous one or from the new snapshot, both will work :)
```

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 3]
```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
...
+The reason we require this is, with every snapshot change, the network might get partitioned, and you may be on a different genesis than the current one.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:66]: I'm not sure the snapshot language is as relevant as it used to be. I believe that now we are at a level of stability where a wipe/purge will only be required in extreme situations. Certainly not as often as earlier testnet stages.

Keen to hear if @nazar-pc has any thoughts on talking about snapshots like this still.

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 4]
```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
...
+In plain English, these commands are like a `reset` button for snapshot changes.
+
+Now follow installation guide.
+
+### Docker Wipe & Purge
+
+In case of Docker setup run `docker compose down -v` (and manually delete custom directories if you have specified them).
+
+Now follow installation guide.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:75]:
```suggestion
Now follow the installation guide.
```

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx [hunk 5]
```diff
@@ -53,6 +53,31 @@ In general you should be able to download the latest release, and re-start the N
...
+
+If you are having some issues with running the node or the farmer for the subspace network, feel free to join our Discord or even better you can take a look at our Forums and review and existing questions or post a new one if needed!
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:77]:
```suggestion
If you are having issues with running a node or farmer for Subspace, feel free to join our Discord or even better you can take a look at our forums and review existing questions or post a new one if needed!
```

File: docs/farming-&-staking/farming/advanced-cli/cli-troubleshooting.mdx [hunk 6]
```diff
@@ -15,39 +15,10 @@ keywords:
 If you are facing issues with your node/farmer you can try a few of the following things below, if you are unable to get your issue resolved please check our [Forums](https://forums.subspace.network) to see if your issue may have been solved, if its a new one feel free to post it! You can also join our [Discord](https://discord.gg/subspace-network) for additional Peer to Peer help.
 
 :::info
-
 We have included a few tutorials below that may help you in your support journey, this is not an all inclusive list but we welcome contributions
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/advanced-cli/cli-troubleshooting.mdx:18]:
```suggestion
We have included a few tutorials below that may help you in your support journey, this is not an all inclusive list and we welcome contributions
```

File: docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx [hunk 7]
```diff
@@ -17,6 +17,36 @@ import TabItem from '@theme/TabItem';
...
 
+### Switching to a new release
+
+:::info
+Unless specifically mentioned by the Development team you should **NOT** have to wipe your node and configuration on new releases.
+:::
+
+In general you should be able to download the latest release of **Pulsar**, and re-start with the same commands as you started to prior version with no errors.
+
+There are some cases where version updates will cause issue with **Pulsar** and you may have to use the `./pulsar_release wipe` command, typically when errors occur.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx:28]:
```suggestion
There are rare cases where version updates will cause issues with **Pulsar** and you may have to use the `./pulsar_release wipe` command, typically when errors occur. 
```

File: docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx [hunk 8]
```diff
@@ -17,6 +17,36 @@ import TabItem from '@theme/TabItem';
...
+### Switching to a new release
+
+:::info
+Unless specifically mentioned by the Development team you should **NOT** have to wipe your node and configuration on new releases.
+:::
+
+In general you should be able to download the latest release of **Pulsar**, and re-start with the same commands as you started to prior version with no errors.
+
+There are some cases where version updates will cause issue with **Pulsar** and you may have to use the `./pulsar_release wipe` command, typically when errors occur. 
+If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) Server to ask for help.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx:29]:
```suggestion
If you have any issues you can always check our [Forums](https://forums.subspace.network) and hop in our [Discord](https://discord.gg/subspace-network) server to ask for help.
```

File: docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx [hunk 9]
```diff
@@ -17,6 +17,36 @@ import TabItem from '@theme/TabItem';
...
+
+### Wipe Node & Farmer
+
+:::tip Updated from a previous version and now having issues?
+
+Occasionally after updating to a new version of the Pulsar you will need to wipe your node and farmer, generally this should not be required but can be attempted if your farmer is having issues after having had worked fine previously.
```
Comment (jim-counter) [inline docs/farming-&-staking/farming/pulsar/pulsar-tips.mdx:35]:
```suggestion
Occasionally after updating to a new version of Pulsar you will need to wipe your node and farmer, generally this should not be required but it can be attempted if your farmer is having issues after having been working fine previously. Particularly if there is suspected data corruption e.g. after a power outage.
```

**Review by jim-counter — CHANGES_REQUESTED** [review]
Thanks for picking this up Emil. Suggested some edits.

File: docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx (hunk 3 above)
Comment (nazar-pc) [reply docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:66]: I agree, during whole 3g there were there were just a few one-offs where specific farmers had to wipe things. For everyone else there should be no issues that require starting from scratch and only transitioning to the next major version of the network requires this.

Comment (jim-counter) [reply docs/farming-&-staking/farming/advanced-cli/cli-tips.mdx:66]: Let's reserve wipe/purge for switching networks or recovering from catastrophic data corruption. The former should be non-existent after mainnet but possibly useful for parallel testnets. The latter will hopefully be rare but there could be users with stability issues it will help.

**Comment by EmilFattakhov** [comment]
Thanks, please another glance when you get a chance @jim-counter!

**Review by jim-counter — APPROVED** [review]
LGTM!
